<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>CustomizeCannyDetection</class>
 <widget class="QMainWindow" name="CustomizeCannyDetection">
  <property name="windowModality">
   <enum>Qt::WindowModal</enum>
  </property>
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>547</width>
    <height>317</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Customize Canny Detection</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QPushButton" name="butViewCannyImage">
    <property name="geometry">
     <rect>
      <x>200</x>
      <y>50</y>
      <width>191</width>
      <height>41</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Times New Roman</family>
      <pointsize>16</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="text">
     <string>View Canny Image</string>
    </property>
   </widget>
   <widget class="QSlider" name="sliderMinThreshold">
    <property name="geometry">
     <rect>
      <x>200</x>
      <y>180</y>
      <width>341</width>
      <height>22</height>
     </rect>
    </property>
    <property name="maximum">
     <number>255</number>
    </property>
    <property name="value">
     <number>0</number>
    </property>
    <property name="orientation">
     <enum>Qt::Horizontal</enum>
    </property>
   </widget>
   <widget class="QSlider" name="sliderMaxThreshold">
    <property name="geometry">
     <rect>
      <x>200</x>
      <y>240</y>
      <width>341</width>
      <height>22</height>
     </rect>
    </property>
    <property name="maximum">
     <number>255</number>
    </property>
    <property name="value">
     <number>0</number>
    </property>
    <property name="orientation">
     <enum>Qt::Horizontal</enum>
    </property>
   </widget>
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>170</y>
      <width>121</width>
      <height>41</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Algerian</family>
      <pointsize>12</pointsize>
     </font>
    </property>
    <property name="text">
     <string>MinThreshold</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_2">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>230</y>
      <width>131</width>
      <height>41</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Algerian</family>
      <pointsize>12</pointsize>
     </font>
    </property>
    <property name="text">
     <string>MaxThreshold</string>
    </property>
   </widget>
   <widget class="QLabel" name="labelMinThreshold">
    <property name="geometry">
     <rect>
      <x>140</x>
      <y>170</y>
      <width>31</width>
      <height>41</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Algerian</family>
      <pointsize>12</pointsize>
     </font>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="labelMaxThreshold">
    <property name="geometry">
     <rect>
      <x>140</x>
      <y>230</y>
      <width>31</width>
      <height>41</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Algerian</family>
      <pointsize>12</pointsize>
     </font>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
